core/domain/topic_services.py

```diff
@@ -347,20 +529,22 @@ def check_can_edit_topic(user, topic_rights):
     return False
 
 
-def assign_role(committer, assignee_id, new_role, topic_id):
+def assign_role(committer, assignee, new_role, topic_id):
```

Comment:
Why this change, if you don't need anything besides the assignee ID?

In general, pass in only what is needed.

Reply:
I added another check here to make sure that the assignee has the required action to be made a manager. For that, the UserActionsInfo object was required.